src/lib/consensus/intf.ml

```diff
@@ -413,6 +413,9 @@ module type S = sig
         -> Host_and_port.t Rpc.Implementation.t list
     end
 
+    (* Check whether we are in the genesis epoch *)
+    val is_genesis : Coda_base.Block_time.t -> bool
```

Comment:
Can we find a more abstract name for this? Ideally, we are trying to keep the consensus interface generic over what consensus system we are using, so I'm trying to avoid referencing stuff like `slot` or `epoch` in the interface. Perhaps we could roll this logic up into `should_bootstrap` or something and try and use that hook in a way that would prevent us from bootstrapping at start up if we are still in genesis?

Reply:
`should_bootstrap` takes a look at the candidate transition and decide whether to bootstrap or not. But we need to decide to bootstrap or not at the very start before we receive the first transition. This is done by `is_genesis` function.

Reply:
would still prefer to find a more generalized name than `is_genesis`, but let's just merge this and get this tested for now